nullaway/src/test/java/com/uber/nullaway/NullAwaySerializationTest.java

```diff
@@ -41,21 +52,70 @@
 /** Unit tests for {@link com.uber.nullaway.NullAway}. */
 @RunWith(JUnit4.class)
 public class NullAwaySerializationTest extends NullAwayTestsBase {
-
-  private SerializationTestHelper serializationTestHelper;
   private String configPath;
+  private Path root;
```

Comment:
There are rather massive changes to this class now that seem independent of field init serialization.  Would it be possible to do these changes in a separate PR?  That separate PR could land before this one, maybe?  If the changes are very intertwined, can you explain more what is going on with these changes?

Reply:
Sure, I created a separate [PR](https://github.com/uber/NullAway/pull/579) which updates the implementation of the test tools. If you think this new implementation is not necessary and testing is not required, please let me know, and I will close the PR and switch back to the current version. Thank you very much.